docs/algorithms/theory-idars.rst

```diff
@@ -0,0 +1,114 @@
+IDaRS Theory
+====================
+
+IDaRS stands for **I**\ terative **D**\ raw **a**\ nd **R**\ ank **S**\ ampling,
+an algorithm introduced in an article
+`[Read/download]
+<https://www.thelancet.com/journals/land$ig/article/PIIS2589-7500(2100180-1/fulltext>`_
+by **Bilal et al.**, *"Development and validation of a weakly supervised
+deep learning framework to predict the status of molecular pathways
+and key mutations in colorectal cancer from routine histology images:
+a retrospective study"*.
```

Comment:
Shall we add doi as well?

Reply:
Yes. 
DOI: https://doi.org/10.1016/S2589-7500(21)00180-1

Reply:
Everything is fine. Since @shaneahmed  merged with develop, I have fixed one typo on the branch doc-idars-alg. Changed " nd " to " and "? Can this be merged with develop?

Reply:
This is how it looks like on readthedocs https://tia-toolbox.readthedocs.io/en/doc-idars-alg/algorithms/theory-idars.html

@DavidBAEpstein if you are happy with this, I will approve and merge this PR.

Reply:
Thanks David. Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/383/commits/a5587957203a3ea43c486088ac948f9c5eac09bc